pydra/engine/tests/test_workflow.py

```diff
@@ -4687,3 +4687,33 @@ def one_arg_inner(start_number):
 
     res = test_outer.result()
     assert res[0].output.res2 == 23 and res[1].output.res2 == 23
+
+
+def test_rerun_errored(tmpdir, capfd):
+    @mark.task
+    def pass_odds(x):
+        if x % 2 == 0:
+            print(f"{x}%2 = {x % 2} (error)")
+            raise Exception("even error")
+        else:
+            print(f"x%2 = {x % 2}")
+            return x
+
+    wf = Workflow(name="wf", input_spec=["x"], cache_dir=tmpdir)
+    wf.add(pass_odds(name="pass_odds", x=[1, 2, 3, 4, 5]).split("x"))
+    wf.set_output([("out", wf.pass_odds.lzout.out)])
+
+    with pytest.raises(Exception) as exinfo:
+        wf()
+    print("...")
```

Comment:
remove print.

Reply:
I kept the print("...") just to make a distinction in the stdout of running the task the first time versus the second time. print("...") could be removed, but I based the assert statements on stdout, so the assert statements would have to be adjusted.